src/plugins/abrt-dump-xorg.c

```diff
@@ -16,6 +16,7 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+#include <glib-2.0/glib/gstdio.h>
```

Comment:
This is not what the path should be. You got it right (almost) everywhere else, so I imagine you just need to adjust the build definitions.